### ISSUE TITLE
Crash on Windows 10 x64

### ISSUE THREAD

**Issue by megacoderencoderdecoder** [DynamoRIO/drmemory#2257]
```
C:\Projects\ToolsProjects\TestRetdec\x64\Debug>c:\Users\Karaul0v\Downloads\DrMemory-Windows-2.3.0-1\bin\drmemory.exe C:\Users\Karaul0v\Downloads\retdec-master-windows-64b\retdec\bin\retdec-llvmir2hll.exe -target-hll=c -output-format=plain -var-renamer=readable -var-name-gen=fruit -var-name-gen-prefix= -call-info-obtainer=optim -arithm-expr-evaluator=c -validate-module -o C:\Projects\ToolsProjects\TestRetdec\x64\Debug\TestRetdec2.exe.c C:\Projects\ToolsProjects\TestRetdec\x64\Debug\TestRetdec2.exe.bc -enable-debug -emit-debug-comments -config-path=C:\Projects\ToolsProjects\TestRetdec\x64\Debug\TestRetdec2.exe.config.json -max-memory-half-ram
~~Dr.M~~ Dr. Memory version 2.3.0
~~Dr.M~~ Running "C:\Users\Karaul0v\Downloads\retdec-master-windows-64b\retdec\bin\retdec-llvmir2hll.exe -target-hll=c -output-format=plain -var-renamer=readable -var-name-gen=fruit -var-name-gen-prefix= -call-info-obtainer=optim -arithm-expr-evaluator=c -validate-module -o C:\Projects\ToolsProjects\TestRetdec\x64\Debug\TestRetdec2.exe.c C:\Projects\ToolsProjects\TestRetdec\x64\Debug\TestRetdec2.exe.bc -enable-debug -emit-debug-comments -config-path=C:\Projects\ToolsProjects\TestRetdec\x64\Debug\TestRetdec2.exe.config.json -max-memory-half-ram"
~~Dr.M~~ Using system call file c:\Users\Karaul0v\Downloads\DrMemory-Windows-2.3.0-1\drmemory\logs\symcache\syscalls_x64.txt
<Application C:\Users\Karaul0v\Downloads\retdec-master-windows-64b\retdec\bin\retdec-llvmir2hll.exe (3800).  Dr. Memory internal crash at PC 0x00000000710cbc4b.  Please report this at http://drmemory.org/issues along with the results of running '-debug -dr_debug'.  Program aborted.
0xc0000005 0x00000000 0x00000000710cbc4b 0x00000000710cbc4b 0x0000000000000001 0x000000007118dcbc
Base: 0x0000000071000000
Registers: eax=0x0000000000000001 ebx=0x00000231b783d3c0 ecx=0xffffffffffffffff edx=0x0000000000000001
        esi=0x00000231b643ed70 edi=0x0000000000000000 esp=0x00000231b643ec40 ebp=0x0000000000000000
        r8 =0x0000000000000000 r9 =0x0000000000000000 r10=0x0000000000000000 r11=0x0000000000000246
        r12=0x0000000000000001 r13=0x00000231b49c4048 r14=0x0000000000000000 r15=0x0000000000000000
        eflags=0x0000000000010286
2.3.0-1-(Feb  6 2020 06:07:09) WinVer=105;Rel=1903;Build=18362;Edition=Professional
-no_dynamic_options -disasm_mask 8 -logdir 'c:\Users\Karaul0v\Downloads\DrMemory-Windows-2.3.0-1\drmemory\logs\dynamorio' -client_lib 'c:\Users\Karaul0v\Downloads\DrMemory-Windows-2.3.0-1\bin64\release\drmemorylib.dll;0;-logdir `c:\Users\Karaul0v\Downloads\DrMemory-Windows-2.3.0-1\drmemory\logs` -symcache_dir `c:\Users\K>
~~Dr.M~~ WARNING: application exited with abnormal code 0xffffffff
```

-```
--------------------------
Dr. Memory Notice: C:\Users\Karaul0v\Downloads\retdec-master-windows-64b\retdec\bin\retdec-llvmir2hll.exe(32796)
---------------------------
Application C:\Users\Karaul0v\Downloads\retdec-master-windows-64b\retdec\bin\retdec-llvmir2hll.exe (32796).  Dr. Memory internal crash at PC 0x00000000710cbc4b.  Please report this at http://drmemory.org/issues along with the results of running '-debug -dr_debug'.  Program aborted.
0xc0000005 0x00000000 0x00000000710cbc4b 0x00000000710cbc4b 0x0000000000000001 0x000000007118dcbc
Base: 0x0000000071000000
Registers: eax=0x0000000000000001 ebx=0x000001aa1fded3c0 ecx=0xffffffffffffffff edx=0x0000000000000001
	esi=0x000001aa1e9eed70 edi=0x0000000000000000 esp=0x000001aa1e9eec40 ebp=0x0000000000000000
	r8 =0x0000000000000000 r9 =0x0000000000000000 r10=0x0000000000000000 r11=0x0000000000000246
	r12=0x0000000000000001 r13=0x000001aa1cf64048 r14=0x0000000000000000 r15=0x0000000000000000
	eflags=0x0000000000010286
2.3.0-1-(Feb  6 2020 06:07:09) WinVer=105;Rel=1903;Build=18362;Edition=Professional
-no_dynamic_options -disasm_mask 8 -logdir 'c:\Users\Karaul0v\Downloads\DrMemory-Windows-2.3.0-1\drmemory\logs\dynamorio' -client_lib 'c:\Users\Karaul0v\Downloads\DrMemory-Windows-2.3.0-1\bin64\release\drmemorylib.dll;0;-logdir `c:\Users\Karaul0v\Downloads\DrMemory-Windows-2.3.0-1\drmemory\logs` -symcache_dir `c:\Users\K
---------------------------
ОК   
---------------------------

```
Console app, i got error when conhost.exe terminated. But target process not conhost.exe.....


Sorry for bad english languange!

**Comment by jakka30** [comment]
I too got crash with x64 application on Windows 10.

![image](https://user-images.githubusercontent.com/6943689/74472807-2145a880-4e68-11ea-999d-8e966568a69f.png)

<Application Q:\binx64\Debug\LP360.exe (11616).  Dr. Memory internal crash at PC 0x00000000710cbc4b.  Please report this at http://drmemory.org/issues along with the results of running '-debug -dr_debug'.  Program aborted.
0xc0000005 0x00000000 0x00000000710cbc4b 0x00000000710cbc4b 0x0000000000000001 0x000000007118dcbc
Base: 0x0000000071000000
Registers: eax=0x0000000000000001 ebx=0x000002842d15ca40 ecx=0xffffffffffffffff edx=0x0000000000000001
        esi=0x000000e313bfed78 edi=0x0000000000000000 esp=0x000000e313bfec38 ebp=0x0000000000000000
        r8 =0x0000000000000000 r9 =0x0000000000000000 r10=0x0000000000000000 r11=0x0000000000000246
        r12=0x0000000000000001 r13=0x0000000000000000 r14=0x000000e313bff300 r15=0x0000000000000000
        eflags=0x0000000000010286
2.3.0-1-(Feb  6 2020 06:07:09) WinVer=105;Rel=1903;Build=18362;Edition=Enterprise
-no_dynamic_options -disasm_mask 8 -logdir 'C:\Users\sjakka\AppData\Roaming\Dr. Memory\dynamorio' -client_lib 'C:\Program Files (x86)\Dr. Memory\bin64\release\drmemorylib.dll;0;-logdir `C:\Users\sjakka\AppData\Roaming\Dr. Memory` -symcache_dir `C:\Users\sjakka\AppData\Roaming\Dr. Memory\symcache` -lib_blacklist `C:\WINDO>
~~Dr.M~~ WARNING: application exited with abnormal code 0xffffffff

**Next tried with -ignore_kernel option. It crashed again.**
<Application Q:\binx64\Debug\LP360.exe (27876).  Dr. Memory internal crash at PC 0x00000000710cbc4b.  Please report this at http://drmemory.org/issues along with the results of running '-debug -dr_debug'.  Program aborted.
0xc0000005 0x00000000 0x00000000710cbc4b 0x00000000710cbc4b 0x0000000000000001 0x000000007118dcbc
Base: 0x0000000071000000
Registers: eax=0x0000000000000001 ebx=0x000001bbb70eca40 ecx=0xffffffffffffffff edx=0x0000000000000001
        esi=0x0000006b8b1ff178 edi=0x0000000000000000 esp=0x0000006b8b1ff038 ebp=0x0000000000000000
        r8 =0x0000000000000000 r9 =0x0000000000000000 r10=0x0000000000000000 r11=0x0000000000000246
        r12=0x0000000000000001 r13=0x0000000000000000 r14=0x0000006b8b1ff700 r15=0x0000000000000000
        eflags=0x0000000000010286
2.3.0-1-(Feb  6 2020 06:07:09) WinVer=105;Rel=1903;Build=18362;Edition=Enterprise
-no_dynamic_options -disasm_mask 8 -logdir 'C:\Users\sjakka\AppData\Roaming\Dr. Memory\dynamorio' -client_lib 'C:\Program Files (x86)\Dr. Memory\bin64\release\drmemorylib.dll;0;`-ignore_kernel` -logdir `C:\Users\sjakka\AppData\Roaming\Dr. Memory` -symcache_dir `C:\Users\sjakka\AppData\Roaming\Dr. Memory\symcache` -lib_bl>
~~Dr.M~~ WARNING: application exited with abnormal code 0xffffffff

**Next tried with -debug -dr_debug option and crashed it too.**
~~Dr.M~~ ASSERT FAILURE (thread 27564): D:\drmemory_package\drmemory\fastpath_x86.c:1004: mi->memsz <= 4 (load2x of 8-byte memop not supported)
![image](https://user-images.githubusercontent.com/6943689/74473194-ce202580-4e68-11ea-8082-5a22c6c66df9.png)

Thanks,
Suresh

**Comment by derekbruening** [comment]
There is not enough information here to act on.  Please use the bug template which is supposed to be required.  There is a Github bug (waiting on them to fix it) where if you're not logged in at the time you click "create new issue" and only log in then it will bypass the template: is that how this bug was filed without the template?

The key part of the template is providing a method to reproduce, and helping to drill down on which part of Dr. Memory contains the problem:

**To Reproduce**
Steps to reproduce the behavior:
1. Pointer to a minimized application (ideally the source code for it and instructions on which toolchain it was built with).
2. Precise command line for running the application.
3. Exact output or incorrect behavior.

Please also answer these questions drawn from  https://github.com/DynamoRIO/drmemory/wiki/Debugging#narrowing-down-the-source-of-the-problem :
 - Does the problem go away when running in light mode (pass `-light` to Dr. Memory)?
 - Does the problem go away when running with the options `-leaks_only -no_count_leaks -no_track_allocs`?
 - Does the problem go away when running under plain DynamoRIO?  Do this by running `dynamorio/bin32/drrun -- <application and args>` or `dynamorio/bin64/drrun -- <application and args>` depending on the bitwidth of your applicaiton.  (Ignore warnings about "incomplete installation".)

**Comment by derekbruening** [comment]
The debug build assert above makes this look like a duplicate of #2183:
```
ASSERT FAILURE (thread 27564): D:\drmemory_package\drmemory\fastpath_x86.c:1004: mi->memsz <= 4 (load2x of 8-byte memop not supported)
```

However, again, some way to reproduce, or identifying the precise instruction sequence that led to the assert, is needed.
